package.json

```diff
@@ -46,12 +46,12 @@
     "karma-sourcemap-loader": "^0.3.7",
     "karma-webpack": "^2.0.2",
     "sinon": "^1.17.7",
-    "underscore": "^1.13.1",
+    "underscore": "^1.13.6",
```

Comment:
Technically we wouldn't have had to change this, just running `yarn install` would install the latest stable minor version when we have the caret `^` but this is fine too 👍

Reply:
Updated to ^1.13.0 so we have the same in Mango, Sidecar and Ventana